go.mod

```diff
@@ -6,9 +6,9 @@ require (
 	github.com/kubernetes-csi/csi-lib-utils v0.15.0-rc0.0.20230811202321-0f6938005f63
```

Comment:
This is still rc,  we need to update when csi-lib-utils v0.15.0 is ready

Reply:
Updated `csi-lib-utils v0.15.0`